src/App.tsx

```diff
@@ -15,48 +17,119 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortField {
```

Comment:
```suggestion
enum SortType {
```

Reply:
<img width="748" alt="image" src="https://github.com/mate-academy/react_list-of-goods-hooks/assets/86789161/1af367ca-92f3-4dbf-8d5a-5ff1c3239ece">